scalikejdbc-streams/src/main/scala/scalikejdbc/streams/StreamReadySQL.scala

```diff
@@ -15,9 +15,9 @@ case class StreamReadySQL[A] private (
 
   private[streams] lazy val extractor: (WrappedResultSet) => A = underlying.extractor
 
-  private[streams] lazy val statement: String = underlying.statement
-  private[streams] lazy val rawParameters: Seq[Any] = underlying.rawParameters
-  private[streams] lazy val parameters: Seq[Any] = underlying.parameters
+  private[streams] def statement: String = underlying.statement
```

Comment:
Is there any specific reason to change only these three attributes (statement, rawParameters, and parameters)? What do you think changing extractor, fetchSize, tags, and queryTimeout as well?